### PR TITLE
CHANGES.md for Chapel 2.1

### PR DESCRIPTION
**Pull request by bradcray**
This is the list of (notable) changes made for Chapel 2.1—thanks everyone!

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,9 +1,519 @@
...
+
+Runtime Library Changes
+-----------------------
+* updated the runtime to use the GASNet-EX API by default for `CHPL_COMM=gasnet`
+* updated GASNet to use the same copy of `hwloc` as Chapel does
+* added support for transparent huge pages with OFI  
+  (see https://chapel-lang.org/docs/2.1/platforms/libfabric.html#transparent-hugepages)
+* ensured that co-locales respect `CHPL_RT_MAX_HEAP_SIZE` 
+  (see https://chapel-lang.org/docs/2.1/platforms/libfabric.html#memory-registration-and-the-heap)
+* added the ability to enable shared-memory bypass when using 'gasnet/ofi'
```
Comment (bonachea) [inline CHANGES.md:354]:
```suggestion
* added the ability to enable shared-memory bypass when using 'gasnet/ibv'
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,9 +1,519 @@
...
+* added the ability to use a dedicated core for progress threads with 'gasnet'
```
Comment (bonachea) [inline CHANGES.md:355]:
```suggestion
* added the ability to use a dedicated core for progress threads with 'gasnet'
* added the ability to enable a blocking send thread for `gasnet/ibv`
```

**Review by bonachea — CHANGES_REQUESTED** [review]

File: CHANGES.md [hunk 3]
```diff
@@ -1,9 +1,519 @@
...
+   and https://chapel-lang.org/docs/2.1/modules/standard/List.html#List.warnForListParsafeMismatch)
+
+Launchers
+---------
+* updated the `pbs-gasnetrun_ibv` launcher to request exclusive access to nodes
+
+Runtime Library Changes
+-----------------------
+* updated the runtime to use the GASNet-EX API by default for `CHPL_COMM=gasnet`
+* updated GASNet to use the same copy of `hwloc` as Chapel does
```
Comment (bonachea) [inline CHANGES.md:349]: Suggest rewording this slightly, to reflect the fact that previously Chapel was force-disabling GASNet's hwloc support

```suggestion
* enable GASNet use of `hwloc`, with the same copy as Chapel
```

File: CHANGES.md (hunk 2 above)
Comment (bradcray) [reply CHANGES.md:318]: Thanks!

File: CHANGES.md (hunk 3 above)
Comment (bradcray) [reply CHANGES.md]: Thanks for clarifying.  I took a slightly different wording, but hopefully captured the change accurately:

```
* enabled GASNet's `hwloc` support, using the same copy of `hwloc` as Chapel
```

**Comment by bradcray** [comment]
I missed the DCO sign-off when merging Dan's GitHub-based suggestion, so backed out and re-implemented those changes and force-pushed the results.

**Review by bonachea — APPROVED** [review]
LGTM!
